lettermgmt/models/__init__.py

```diff
@@ -18,12 +17,10 @@
 #    You should have received a copy of the GNU Affero General Public License
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
-###############################################################################
-
+##############################################################################
 from . import letter_class
 from . import letter_folder
 from . import res_letter
 from . import letter_type
 from . import letter_reassignment
-from . import letter_history
 from . import letter_channel
```

Comment:
I disagree. When importing several things from the same module, it is a usual practice to group them, but import of  different modules is used to be on several lines.